client/src/modules/dashboard/Events/components/SendEmailModal.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button } from '@chakra-ui/button';
+import { Checkbox } from '@chakra-ui/checkbox';
+import { Stack } from '@chakra-ui/layout';
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalFooter,
+  ModalOverlay,
+} from '@chakra-ui/modal';
+import { Alert, AlertIcon, AlertDescription } from '@chakra-ui/react';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useSendEventInviteMutation } from 'generated/graphql';
+interface SendEmailModalProps {
+  onClose: () => any;
+  isOpen: boolean;
+  eventId: number;
+}
+const SendEmailModal: React.FC<SendEmailModalProps> = ({
+  onClose,
+  isOpen,
+  eventId,
+}) => {
+  const {
+    register,
+    getValues,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const checkAtLeastOne = () => {
+    return getValues('confirmed') ||
+      getValues('on_waitlist') ||
+      getValues('canceled')
+      ? true
+      : 'Please tell me if this is too hard.';
+  };
+
+  const [publish] = useSendEventInviteMutation();
+  const onSubmit = (data: any) => {
```

Comment:
Could you add a type for this?

```ts
interface FormInputs {
  confirmed: boolean;
  on_waitlist: boolean;
  canceled: boolean;
}
```
should work just fine.

Reply:
Added the above interface to SendEmailModal.tsx